rdr_service/dao/ppsc_dao.py

```diff
@@ -22,5 +24,16 @@ def __init__(self, model_type):
     def from_client_json(self):
         pass
 
+    def to_client_json(self, payload):
+        # pylint: disable=unused-argument
+        return "Event Record Created"
```

Comment:
Should we return the participant Id with this?
"Event record created for: P<pid>"

Reply:
Good suggestion. I'll add that.